File: examples-sdk/typescript/508-villog-bench-multithread.ts
#!/usr/bin/env tsx
// 508-villog-bench-multithread — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 508-villog-bench-multithread.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/508-villog-bench-multithread.py
#!/usr/bin/env python3
"""508-villog-bench-multithread — Python SDK equivalent
Compile: vil compile --from python --input 508-villog-bench-multithread.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
